<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smu_search_dialog</class>
 <widget class="QDialog" name="smu_search_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SMU Connections</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="smu_connections">
     <property name="title">
      <string>Connections</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QPushButton" name="smu_identify">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Identify</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QListWidget" name="connection_list"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="smu_disconnect">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Connect to Sourcemeters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QPlainTextEdit" name="search_log">
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="search_progress"/>
      </item>
      <item>
       <widget class="QPushButton" name="search">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Search for SMUs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>smu_search_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
